node/src/rpc.rs

```diff
@@ -50,11 +48,9 @@ use crate::tracing;
 /// A type representing all RPC extensions.
 pub type RpcExtension = jsonrpsee::RpcModule<()>;
 
-#[reuse(evm, no_evm)]
-pub mod __ {
+pub mod evm {
```

Comment:
Shall we simply remove this mod scope?

Reply:
let's save it for this round? since if we want to revert, I hate to handle the whitespace error by clippy.